src/partials/layout.html

```diff
@@ -42,6 +42,17 @@
     ></script>
   </head>
   <body data-page="{{ page }}">
+    <!-- top-bar -->
+    <div class="flex items-center justify-center text-xs text-gray-700 bg-blue-200 h-8">
+      &#127881; Livebook v0.5 is out.&nbsp 
+      <a
+        class="border-b border-gray-700"
+        href="https://news.livebook.dev/v0.5-flowcharts-custom-widgets-intellisense-and-ui-improvements-4FcvbG"
+        target="_blank"
+        rel="noopener"
+        >Read the announcement</a
+      >.
```

Comment:
It's better to use padding rather than fixed height, this way if we have a longer notification, or the screen is very small, the text will wrap naturally and keep reasonable spacing :)

```suggestion
    <!-- Notification top bar -->
    <div class="p-2 text-center text-xs text-gray-700 bg-blue-200">
      &#127881; Livebook v0.5 is out.
      <a
        class="font-medium border-b border-gray-700"
        href="https://news.livebook.dev/v0.5-flowcharts-custom-widgets-intellisense-and-ui-improvements-4FcvbG"
        target="_blank"
        rel="noopener"
        >Read the announcement</a
      >.
```